packages/KDropdownMenu/KDropdownItem.vue

```diff
@@ -1,42 +1,47 @@
 <template>
-  <li
-    :class="{
-      'has-divider': type !== 'link' && hasDivider,
-      'disabled': type === 'default' && disabled,
-      'danger': isDangerous,
-      'k-dropdown-selected-option': selected
-    }"
-    :data-testid="`k-dropdown-item-${label.replace(' ', '-')}`"
-    class="k-dropdown-item w-100"
-  >
-    <router-link
-      v-if="type === 'link' && to"
-      :to="!disabled ? to : $router.currentRoute.path"
-      :class="{ 'disabled': disabled, 'has-divider': hasDivider }"
-      class="k-dropdown-item-trigger"
-      data-testid="k-dropdown-item-trigger"
-    >
-      <slot>{{ label }}</slot>
-    </router-link>
-    <KButton
-      v-else-if="type === 'button'"
-      :disabled="disabled"
-      :is-rounded="false"
-      class="k-dropdown-item-trigger btn-link k-button non-visual-button"
-      data-testid="k-dropdown-item-trigger"
-      v-on="listeners"
-      @click="handleClick"
-    >
-      <slot>{{ label }}</slot>
-    </KButton>
+  <div class="w-100">
```

Comment:
I see a `<div>` as the first element inside `<ul>` tag. Is it semantically correct?

Reply:
Nope :trollface: not valid

Reply:
Fixed, thank @adamdehaven for helping me with the CSS!